lib/remap-functions/src/get_env_var.rs

```diff
@@ -0,0 +1,86 @@
+use remap::prelude::*;
+
+#[derive(Clone, Copy, Debug)]
+pub struct GetEnvVar;
+
+impl Function for GetEnvVar {
+    fn identifier(&self) -> &'static str {
+        "get_env_var"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "name",
+            accepts: |v| matches!(v, Value::Bytes(_)),
+            required: true,
+        }]
+    }
+
+    fn compile(&self, mut arguments: ArgumentList) -> Result<Box<dyn Expression>> {
+        let name = arguments.required("name")?.boxed();
+
+        Ok(Box::new(GetEnvVarFn { name }))
+    }
+}
+
+#[derive(Debug, Clone)]
+struct GetEnvVarFn {
+    name: Box<dyn Expression>,
+}
+
+impl GetEnvVarFn {
+    #[cfg(test)]
+    fn new(name: Box<dyn Expression>) -> Self {
+        Self { name }
+    }
+}
+
+impl Expression for GetEnvVarFn {
+    fn execute(&self, state: &mut state::Program, object: &mut dyn Object) -> Result<Value> {
+        let bytes = self.name.execute(state, object)?.try_bytes()?;
+        let name = String::from_utf8_lossy(&bytes);
+
+        let value = std::env::var(name.as_ref())
+            .map_err(|_| Error::Call(format!("Environment variable {:?} not found.", name)))?;
```

Comment:
```suggestion
            .map_err(|_| Error::Call(format!(r#"environment variable "{}" not found"#, name)))?;
```

Reply:
Just realized that there are two possible errors:
- Environment variable is not present
- Environment variable is not valid unicode

So I'm using `Error::Call(e.to_string())` instead.